preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -591,7 +591,9 @@ def process_single(  # noqa: C901
         unaligned_nucleotide_sequences = unprocessed.unalignedNucleotideSequences
 
     for segment in config.nucleotideSequences:
-        sequence = unaligned_nucleotide_sequences[segment]
+        sequence = unaligned_nucleotide_sequences.get(segment, None)
```

Comment:
This was only a bug for the not-aligned organism pipeline - I guess I never tested it on a multi-segmented organism.